packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.stories.js

```diff
@@ -0,0 +1,21 @@
+import { storiesOf } from '@storybook/react'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Grommet } from 'grommet'
+import React from 'react'
+
+import readme from './README.md'
+import TranscribedLines from './TranscribedLines'
+
+const config = {
+  notes: {
+    markdown: readme
+  }
+}
+
+storiesOf('TranscribedLines', module)
+
+  .add('default', () => (
+    <Grommet theme={zooTheme}>
```

Comment:
Wrapping this with the `Grommet` component is not necessary for tests unless you are using the `withTheme` or `withThemeContext` HOCs.

Reply:
This is the default produced by the `plop` template.